### PR TITLE
Cherry pick from 61 to 62

### PR DESCRIPTION
**Pull request by mgsergio**

### CODE REVIEW

**Comment by mgsergio** [comment]
DUDE JTTP

**Comment by mgsergio** [comment]
Ошибка в 62, это можно мерджить.

File: indexer/osm_editor.cpp [hunk 1]
```diff
@@ -131,6 +131,13 @@ bool IsObsolete(editor::XMLFeature const & xml, FeatureID const & fid)
   return uploadTime != my::INVALID_TIME_STAMP &&
          my::TimeTToSecondsSinceEpoch(uploadTime) < GetMwmCreationTimeByMwmId(fid.m_mwmId);
 }
+
+m2::PointD GetSomeFeaturePoint(editor::XMLFeature const & xml)
+{
+  if (xml.GetType() == XMLFeature::Type::Node)
```
Comment (ygorshenin) [inline indexer/osm_editor.cpp]: ASSERT that geometry is not empty?

**Comment by ygorshenin** [comment]
LGTM with a nit.

**Comment by mgsergio** [comment]
PTAL & Merge

File: indexer/osm_editor.cpp (hunk 1 above)
Comment (mgsergio) [inline indexer/osm_editor.cpp]: Unused function :)
Thanks for your help with discovering it.

**Comment by Zverik** [comment]
LGTM

**Comment by mpimenov** [comment]
for for for try if goto

LGTM...

**Pull request merged**